shared/DecisionTree/Microsoft.AspNetCore.Routing.DecisionTree.Sources.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.Routing.DecisionTree.Sources</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Components for building a DecisionTree.</description>
+    <tags></tags>
+    <serviceable>true</serviceable>
+    <contentFiles>
+       <files include="contentFiles/cs/**/*.cs" buildAction="Compile" />
+    </contentFiles>
+  </metadata>
+  <files>
+    <file src="**\*.cs" target="contentFiles/cs/netstandard1.0/" />
```

Comment:
Could we keep the slashes the same (forward slashes)? Different in the `contentFiles` node and here.